spec/models/user_spec.rb

```diff
@@ -17,6 +17,29 @@
     end.to change(ActionMailer::Base.deliveries, :count).by(1)
   end
 
+  describe 'password validations' do
+    it 'disallows common words' do
+      user = create(:user)
+      user.password = 'password'
+
+      expect(user.valid?).to eq false
+      expect(user.errors[:password].first).to match('not strong enough')
+    end
+
+    it 'allows long phrases' do
+      user = create(:user)
+      user.password = 'a long password'
+
+      expect(user.valid?).to eq true
+    end
+
+    it 'allows visitors' do
+      user = create(:user, :unconfirmed)
+
+      expect(user.valid?).to eq true
+    end
```

Comment:
I am happy to change the language. I picked `visitor` because it is used extensively elsewhere in the specs (e.g. `spec/features/visitors/sign_up_spec.rb`).